charts/fluentd/values.yaml

```diff
@@ -130,7 +130,8 @@ secrets:
   logzioShippingToken: ""
   logzioListener: ""
 
-secretName: logzio-logs-secret
+defaultSecretName: logzio-logs-secret
+secretName: ""
```

Comment:
Now with your changes this should have a default value, and people who want will be able to override it.